ghost/core/core/server/data/migrations/versions/5.21/2022-10-26-04-50-member-subscription-created-request-id.js

```diff
@@ -0,0 +1,73 @@
+const logging = require('@tryghost/logging');
+const ObjectId = require('bson-objectid').default;
+const {createTransactionalMigration} = require('../../utils');
+const DatabaseInfo = require('@tryghost/database-info');
+
+// This migration links together members_created_events and members_subscription_created_events
+
+module.exports = createTransactionalMigration(
+    async function up(knex) {
+        if (DatabaseInfo.isSQLite(knex)) {
+            logging.info('Skipped linking members_created_events and members_subscription_created_events on SQLite');
+            return;
+        }
+
+        // All events that happened within 15 minutes of each other will be linked
+        const rows = await knex('members_created_events as m')
+            .select('m.id as m_id', 's.id as s_id', 'm.member_id as member_id', 's.subscription_id as subscription_id')
+            .join('members_subscription_created_events AS s', 's.member_id', 'm.member_id')
+            .whereRaw('TIMESTAMPDIFF(MINUTE, s.created_at, m.created_at) between -15 and 15');
+
+        if (!rows.length) {
+            logging.info('Did not find linkable members_created_events and members_subscription_created_events');
+            return;
+        }
+
+        // Attach a unique id to each row
+        for (const row of rows) { // eslint-disable-line no-restricted-syntax
+            row.batch_id = ObjectId().toHexString();
+        }
+
+        // Create batches (insertBatch doesn't support the onConflict option)
+        const batchSize = 1000;
+    
+        for (let i = 0; i < rows.length; i += batchSize) { // eslint-disable-line no-restricted-syntax
+            const batch = rows.slice(i, i + batchSize);
+
+            // Update the members_created_events table using INSERT ON DUPLICATE KEY UPDATE trick
+            const response1 = await knex('members_created_events').insert(batch.map((r) => {
+                return {
+                    id: r.m_id,
+                    batch_id: r.batch_id,
+                    member_id: r.member_id,
```

Comment:
Small suggestion but can you comment the fields that you were forced to include because it's an INSERT vs the ones we actually care about (`id`, `batch_id`)?